projects/demo/src/modules/components/combo-box/examples/7/index.html

```diff
@@ -0,0 +1,28 @@
+<form class="b-form">
+    <tui-combo-box
+        class="b-form"
+        [ngModelOptions]="{standalone: true}"
+        [stringify]="stringify"
+        [(ngModel)]="value"
+    >
+        Find the best employees
+        <input
+            placeholder="Type a name"
+            tuiTextfield
+        />
+        <ng-container *tuiDataList>
+            <tui-data-list>
+                <button
+                    *ngFor="let item of items | tuiFilterByInputWith: stringify"
+                    tuiOption
+                    [disabled]="item.disabled"
+                    [value]="item"
+                >
+                    {{ item | tuiMapper: stringify }}
```

Comment:
Piping is unnecessary for such simple operation.

Reply:
fixed